crypto/objects/obj_dat.c

```diff
@@ -155,6 +155,19 @@ static unsigned long added_obj_hash(const ADDED_OBJ *ca)
     return ret;
 }
 
+/*
+ * Compare two ASN1_OBJECTs, including SNAME and LNAME, but not NIDs.
+ */
+static int obj_equivalent(const ASN1_OBJECT *a, const ASN1_OBJECT *b)
+{
+    return a->length == b->length
+        && memcmp(a->data, b->data, (size_t)a->length) == 0
+        && (a->sn == NULL) == (b->sn == NULL)
+        && strcmp(a->sn ? a->sn : "", b->sn ? b->sn : "") == 0
+        && (a->ln == NULL) == (b->ln == NULL)
+        && strcmp(a->ln ? a->ln : "", b->ln ? b->ln : "") == 0;
```

Comment:
what is everything except the OID is identical?

Reply:
is that in data?

Reply:
yes, my understanding is the the DATA type holds the oid.